ComputerVision/SDK/OCR/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+using Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace ImageOCR
+{
+    class Program
```

Comment:
We should update this to be consistent with the other cases, where it has a Remote and Local example.